test/fx/test_matcher_utils.py

```diff
@@ -141,11 +141,12 @@ def pattern(x, weight):
             return relu, relu_mul_by_two, {"conv": conv, "relu": relu}
 
         from torch._export import capture_pre_autograd_graph
+        from torch.export.wrapper import WrapperModule
```

Comment:
@tugsbayasgalan I remember we wanna remove this wrapper, because it's hard to provide this API that works for all cases. Users probably need to wrap functions themselves.

For example, there might be a free function containing a linear layer without being registered, in this case some user would want a submodule to show up, but today we just make those tensor constants.